test/get_tree_list_use_case_test.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:api/data_source/tree/local_tree_data_source.dart';
+import 'package:api/data_source/tree/remote_tree_data_source.dart';
+import 'package:api/dependency_injection.dart';
+import 'package:api/models/app/tree/tree.dart';
+import 'package:data/data_source/tree/local_tree_data_source_impl.dart';
+import 'package:data/data_source/tree/remote_tree_data_source_impl.dart';
+import 'package:data/local_storage/local_storage.dart';
+import 'package:domain/use_case/tree/get_tree_list_use_case.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mocktail/mocktail.dart';
+
+class MockGetTreeLiseUseCase extends Mock implements GetTreeLiseUseCase {}
+
+class MockLocalStorage extends Mock implements LocalStorage {}
+
+void main() {
+  late MockGetTreeLiseUseCase mockGetTreeLiseUseCase;
+  List<Tree> listTree = [];
+
+  setUpAll(() async {
```

Comment:
Any specific reason that you used 'setUpAll' in this test but 'setUp' in the other ?

Reply:
It's a mistake to the other file, setUpAll is better because it's called just one time. I update that